### PR TITLE
Bug 1106835 - [Settings][RTL] Follow-up bug, Fixed security translations...

### PR DESCRIPTION
**Pull request by EragonJ**
... with a proper way

### CODE REVIEW

**Comment by try-server-hook** [comment]
[EragonJ EragonJ (E.J.) (EragonJ)](https://github.com/EragonJ) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=56353684589c)

File: apps/settings/locales/settings.ar.properties [hunk 1]
```diff
@@ -476,8 +476,8 @@ wifi-join-password=كلمة السّر
 hotspotSettings-header={{hotspotSettings}}
 ssid-name=اسم الشبكة
 hotspot-open=فتح
-hotspot-wpa-psk=WPA (TKIP)
-hotspot-wpa2-psk=WPA2 (AES)
+hotspot-wpa-psk.innerHTML= &#x202a;WPA (TKIP)&#x202c;
```
Comment (zbraniecki) [inline apps/settings/locales/settings.ar.properties]: can you add a Localizator comment here about using &#x202a and &x202c for RTL ?

Comment (stasm) [inline apps/settings/locales/settings.ar.properties]: Better yet, please add the Localization note in settings.en-US.properties.  This is the file that the localizers use as reference.

Comment (EragonJ) [inline apps/settings/locales/settings.ar.properties]: sure ok ! i'll add them in english locale :)

**Comment by try-server-hook** [comment]
[EragonJ EragonJ (E.J.) (EragonJ)](https://github.com/EragonJ) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f6477fb4bc4a)

File: apps/settings/locales/settings.en-US.properties [hunk 2]
```diff
@@ -494,8 +494,22 @@ wifi-join-password=Password
 hotspotSettings-header={{hotspotSettings}}
 ssid-name=SSID Network Name
 hotspot-open=open
-hotspot-wpa-psk=WPA (TKIP)
-hotspot-wpa2-psk=WPA2 (AES)
+
+# LOCALIZATION NOTE
+#
+# Please check settings.ar.properties to see the difference first.
```
Comment (flodolo) [inline apps/settings/locales/settings.en-US.properties]: This doesn't help at all. Localizers won't know what this file is, they're aware of the English file and their own.

Comment (EragonJ) [inline apps/settings/locales/settings.en-US.properties]: Not only localizers would read this file, but if this comment is not understandable for localizers, I can add more.

**Comment by try-server-hook** [comment]
[EragonJ EragonJ (E.J.) (EragonJ)](https://github.com/EragonJ) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=edfc6842f1c7)

**Pull request merged**